src/main/resources/anniv5Resources/summaries/ratings.txt

```diff
@@ -65,6 +65,7 @@ SentinelPack	4	4	2	4	2
 SerpentinePack	4	1	3	2	4
 ShamanPack	4	3	2	1	4
 SilentPack	3	3	4	3	3
+SpheresPack	4	3	1	4	3
```

Comment:
What're the thoughts behind these ratings?
You have 1 defend, 1 weak application and 2 Frost channelers, I agree that that's fits a 3.
You have 3 cards that deal damage outright, none of them very high with the orbs not being super good at dealing damage either. What am I missing for you to rate that a 4?
Considering my prior points, I'm not sure if I agree with the frontload being that high.

Reply:
The three commons are all pretty strong frontload, primarily offensive.
* Sphere is better frontload than you might think as a card that mixes offense and defense, since the orbs it channels are all offensive. Early on, while it's not good against Nob, it's great against Sentries, Laga, and any hallway fight.
* Winterwisp is as high value as Ball Lightning when you have an empty orb slot and still good without one, which is pretty good given that Ball Lightning is one of the best damage commons for any character.
* Dual Blade provides a ton of frontloaded value as soon as you have any orbs. Usually it will be evoking damaging orbs early on.

The uncommons are more of a mixed bag when it comes to offense and frontload, of course, but Ignition is something you will always be happy to take early on and the ignite gives it scaling applications too. The other three you need a critical mass of orbs to reach their full potential.

The early game isn't everything, but frontload from commons is generally how you get through it (plus whatever uncommons you happen to see early), and Spheres has very good options there. Most of them have good utility later on as well -- Sphere retains value through the endgame since it's a decent block card, and while the frontload value of the attacks fades as you get to later acts, the value of the orbs tends to increase. 

I did do five or six playtest runs with the preview version of the pack (before the Frost Conversion and Consolidate changes), so the experience of that is really what the ratings are based on -- what I've written here is my analysis of that experience. To say all of this another way, I came out of the playtesting runs feeling that I'd eagerly take Spheres over the average pack when I was looking to have better frontloaded offense. Whether I'd take it over other packs that also had good frontloaded offense would depend on more specific considerations.